.github/workflows/create-channel.yml

```diff
@@ -44,7 +44,9 @@ jobs:
       run: |
         export CHANNEL=${{ steps.tag.outputs.release_tag }}
         mkdir -p build
-        python3 -m zipapp --python="/usr/bin/env python3" --output=build/builder builder
+        mkdir -p staging
```

Comment:
no on L49 we output to build/builder using staging as the directory to create the zipapp from.

zipapp takes the inner directory contents so if you do:

```
python3 -m zipapp --output=build/builder staging
```

It will create a zip archive of everything under `staging`. This is necessary for the modules to work as expected such that you end up with a ziparchive with the following structure:
```
builder/
    __init__.py
    main.py
    ...
```

before we were doing `python3 -m zipapp --output=build/builder builder` but this will leave out the top level `builder` directory/module which screws up the imports. 


We could probably just do this though instead which removes the need for us to do the copy manually or think about it:

```
pip install --target staging .
python3 -m zipapp --python="/usr/bin/env python3" -m "builder.main:main" --output=build/builder staging
```

Same thing without the manual copy and let's python figure out what the package is, but it has additional files compared to what we were doing so I tried to keep it similar. 

This is what you would do if we had dependencies though for example:
```
pip install -r requirements.txt --target staging
```